Resources/Resources.resx

```diff
@@ -966,6 +966,45 @@
   <data name="timerTrigger_schedule_errorText" xml:space="preserve">
     <value>Invalid Cron Expression. Please consult the &lt;a target='_blank' href='https://azure.microsoft.com/en-us/documentation/articles/functions-bindings-timer/'&gt;documentation&lt;/a&gt; to learn more.</value>
   </data>
+    <data name="twilioSms_accountsid_help" xml:space="preserve">
```

Comment:
Only updated the language neutral resource file. I'm assuming the others are generated from this and we can properly fallback if the resource is not found in the culture specific resource files.